server/src/main/java/org/candlepin/model/PoolCurator.java

```diff
@@ -776,4 +838,17 @@ public Pool findDevPool(Consumer consumer) {
         criteria.setMaxResults(1).uniqueResult();
         return (Pool) criteria.uniqueResult();
     }
+
+    public void lock(List<Pool> poolsToLock) {
+        if (poolsToLock.isEmpty()) {
```

Comment:
I have a [similar method](https://github.com/candlepin/candlepin/pull/1073/files#diff-0cac811ddb60a663206629eab27f0c12R511) in #1073 , but there I need to return the pools as well. perhaps we could simply use that method instead of this?